components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2510,12 +2520,19 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
                                 .getText().trim();
             }
             if (openIDConnectConfigElem
-                    .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.OPENID_CONNECT_SKIP_USER_CONSENT)) !=
+                    .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.OPENID_CONNECT_SKIP_LOGIN_CONSENT)) !=
                     null) {
-                openIDConnectSkipUserConsent =
+                openIDConnectSkipLoginConsent = Boolean.parseBoolean(
                         openIDConnectConfigElem.getFirstChildWithName(
-                                getQNameWithIdentityNS(ConfigElements.OPENID_CONNECT_SKIP_USER_CONSENT))
-                                .getText().trim();
+                                getQNameWithIdentityNS(ConfigElements.OPENID_CONNECT_SKIP_LOGIN_CONSENT))
+                                .getText().trim());
+            }
+            if (openIDConnectConfigElem.getFirstChildWithName(
+                    getQNameWithIdentityNS(ConfigElements.OPENID_CONNECT_SKIP_LOGOUT_CONSENT)) != null) {
```

Comment:
Shouldn't this be backward compatible? That means if 
(ConfigElements.OPENID_CONNECT_SKIP_LOGOUT_CONSENT)) is null, shouldn't we read ConfigElements.OPENID_CONNECT_SKIP_USER_CONSENT from config file and set its value to 'openIDConnectSkipLogoutConsent'?

Reply:
I have added the old configuration as a fallback